api/src/shared/infrastructure/repositories/user-repository.js

```diff
@@ -73,18 +73,12 @@ const getByUsernameOrEmailWithRolesAndPassword = async function (username) {
   return _toDomainFromDTO({ userDTO, membershipsDTO, certificationCenterMembershipsDTO, authenticationMethodsDTO });
 };
 
-const get = function (userId) {
-  return BookshelfUser.where({ id: userId })
-    .fetch()
-    .then((user) => bookshelfToDomainConverter.buildDomainObject(BookshelfUser, user))
-    .catch((err) => {
-      if (err instanceof BookshelfUser.NotFoundError) {
-        throw new UserNotFoundError(`User not found for ID ${userId}`);
-      }
-      throw err;
-    });
+const get = async function (userId) {
+  const foundUser = await knex('users').where('id', userId).first();
+  if (!foundUser) throw new UserNotFoundError(`User not found for ID ${userId}`);
+  return new User(foundUser);
 };
```

Comment:
Aérer le code (même chose pour les fonctions plus bas) 😄 
```suggestion
const get = async function (userId) {
  const foundUser = await knex('users').where('id', userId).first();

  if (!foundUser) throw new UserNotFoundError(`User not found for ID ${userId}`);

  return new User(foundUser);
};

Reply:
cf https://github.com/1024pix/pix/pull/7752#discussion_r1444433692